graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/views/searchtypes/pivot/buckets/OSValuesHandler.java

```diff
@@ -126,7 +126,9 @@ private TermsAggregationBuilder createScriptedTerms(List<? extends BucketSpec> b
 
     private Script scriptForPivots(Collection<? extends BucketSpec> pivots) {
         final String scriptSource = Joiner.on(KEY_SEPARATOR_PHRASE).join(pivots.stream()
-                .map(bucket -> "String.valueOf(doc['" + bucket.field() + "'].size() == 0 ? \"" + MissingBucketConstants.MISSING_BUCKET_NAME + "\" : doc['" + bucket.field() + "'].value)")
+                .map(bucket -> """
+                        String.valueOf((doc.containsKey('%s') && doc['%s'].size() > 0) ? doc['%s'].value : "%s")
+                        """.formatted(bucket.field(), bucket.field(), bucket.field(), MissingBucketConstants.MISSING_BUCKET_NAME))
```

Comment:
is the order of the parameters correct? MISSING_BUCKET at the end while it's not the last parameter?